e2e/test/iothub/method/MethodE2ETests.cs

```diff
@@ -17,8 +18,8 @@ namespace Microsoft.Azure.Devices.E2ETests.Methods
     [TestCategory("IoTHub")]
     public class MethodE2ETests : E2EMsTestBase
     {
-        public static readonly object DeviceResponseJson = new { name = "e2e_test" };
```

Comment:
@drwill-ms The "ServiceSendMethodAndVerifyResponseAsync" method isn't able to make use of these changes because the custom serialization implementation isn't available on the service client yet. So, that assertion will still require us to serialize the payload and then compare.
If we plan on adding custom serialization to the service client eventually, then we'll be able to add this there.